modules/auxiliary/admin/kerberos/get_ticket.rb

```diff
@@ -73,6 +81,33 @@ def initialize(info = {})
   end
 
   def validate_options
+    if datastore['CERT_FILE'].present?
+      certificate = File.read(datastore['CERT_FILE'])
+      begin
+        @pfx = OpenSSL::PKCS12.new(certificate, datastore['CERT_PASSWORD'] || '')
```

Comment:
Minor nitpick, ran into a small issue here when I set `CERT_PASSWORD` when one wasn't required:
<img width="928" alt="image" src="https://user-images.githubusercontent.com/19910435/214594052-14fe8d82-224a-4964-b829-4ae8dc234d61.png">

pretty confusing error message, might be worth bubbling up the message from the exception here

Reply:
That's a good idea. I added the error to the message now. Unfortunately though, when the password is incorrect, it doesn't state that but rather says there was a "mac verify failure". That's still probably more clear than saying it needs to be PKCS12 when it already is.

```
msf6 auxiliary(admin/kerberos/get_ticket) > rerun
[*] Reloading module...
[*] Running module against 192.168.159.10

[-] Auxiliary aborted due to failure: bad-config: Unable to parse certificate file (PKCS12_parse: mac verify failure)
[*] Auxiliary module execution completed
msf6 auxiliary(admin/kerberos/get_ticket) >
```

Reply:
hah yea not ideal, but at least it's something